client/src/components/leftSidebar/index.js

```diff
@@ -8,47 +8,45 @@ import DynamicScatterplot from "../scatterplot/scatterplot";
 import TopLeftLogoAndTitle from "./topLeftLogoAndTitle";
 import AuthN from "./AuthN";
 
-@connect((state) => ({
-  scatterplotXXaccessor: state.controls.scatterplotXXaccessor,
-  scatterplotYYaccessor: state.controls.scatterplotYYaccessor,
-}))
+const LeftSideBar = (props) => {
+  const { scatterplotXXaccessor, scatterplotYYaccessor } = props;
+  const { loading } = useAuth0();
 
+  if (loading) {
+    return <div>Loading...</div>;
+  }
 
-class LeftSideBar extends React.Component {
```

Comment:
Since Auth0 is using hooks, we cannot use Class anymore (this is the main runtime error we're seeing in the console!)